### PR TITLE
assets, dependencies and version expose of fragment

### PR DESCRIPTION
**Pull request by Emre-Kul**
#### What's this PR do?
* assets, dependencies and version exposing of the fragment for client-side rendering.

### CODE REVIEW

File: src/gatewayBFF.ts [hunk 1]
```diff
@@ -365,6 +365,22 @@ export class GatewayBFF {
      */
     private addConfigurationRoute(cb: Function) {
         this.server.addRoute('/', HTTP_METHODS.GET, (req, res) => {
+
+            if(req.query.fragment) {
+                const fragment = this.exposedConfig.fragments[req.query.fragment];
+                if(fragment) {
```
Comment (erdoganoksuz) [inline src/gatewayBFF.ts]: `      if(!fragment) return res.status(404).end();
        return res.status(200).json({
                        assets: fragment.assets,
                        dependencies: fragment.dependencies,
                        version: fragment.version
                    });
                
`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puzzle-microfrontends",
-  "version": "3.26.1",
+  "version": "3.27.1",
```
Comment (Acanguven) [inline package.json]: `3.27.0`

File: src/gatewayBFF.ts (hunk 1 above)
Comment (Emre-Kul) [reply src/gatewayBFF.ts]: 'else' block is unnecessary, I will remove that but I think moving falsy control to the top and removing brackets will not affect readability. I prefer brackets even it is one line for debugging purposes.

File: package.json (hunk 2 above)
Comment (Emre-Kul) [reply package.json]: done

**Pull request merged**